tango_simlib/examples/override_class.py

```diff
@@ -490,3 +496,423 @@ def test_action_stoprainstorm(self, model, tango_dev=None, data_input=None):
         else:
             quant_rainfall.max_bound = float(quant_rainfall.meta['max_bound'])
             quant_rainfall.mean = 1.00
+
+
+class OverrideDish(object):
+
+    def _wait_while_configuring(self):
+        time.sleep(2)
+
+    def action_capture(self, model, tango_dev=None, data_input=None):
```

Comment:
- [ ] Don't raise exceptions if the handler is not implemented, just remove them from the override file so that the default do-nothing implementation can take effect.